docs/Reference/Server.md

```diff
@@ -961,6 +962,7 @@ const defaultRoute = fastify.getDefaultRoute()
 #### setDefaultRoute
 <a id="setDefaultRoute"></a>
 
+**Note**: The default 404 handler, or one set using `setNotFoundHandler`, will never trigger if the default route is overridden. Use [setNotFoundHandler](#setnotfoundhandler) if you want to customize 404 handling instead.
```

Comment:
Please hard wrap lines to be consistent with the rest of the document.

Reply:
done